### PR TITLE
feat: support static action forms with layout

### PR DESCRIPTION
**Pull request by realSpok**
## Definition of Done
#86c097c1w
### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

File: packages/datasource-customizer/test/decorators/actions/collection.test.ts [hunk 1]
```diff
@@ -732,26 +732,6 @@ describe('ActionDecorator', () => {
...
-      });
-
-      expect(newBooks.schema.actions['make photocopy']).toEqual({
-        scope: 'Single',
-        generateFile: false,
-        staticForm: false,
-      });
-    });
-
     test('should compute the form recursively', async () => {
```
Comment (EnkiP) [inline packages/datasource-customizer/test/decorators/actions/collection.test.ts:735]: keep this test with dynamic elements in a row in a page

**Review by EnkiP — CHANGES_REQUESTED** [review]
...

**Review by EnkiP — APPROVED** [review]

**Review by EnkiP — CHANGES_REQUESTED** [review]
sry missclick again :/ need recursive check in getForms

File: packages/datasource-customizer/src/decorators/actions/collection.ts [hunk 2]
```diff
@@ -121,10 +121,9 @@ export default class ActionCollectionDecorator extends CollectionDecorator {
         this.isHandler(form) ||
         form?.some(
           field =>
-            field.type === 'Layout' || // all forms containing some layout elements are handled as dynamic
             Object.values(field).some(value => this.isHandler(value)) ||
```
[the remark this answers is not in the document]
Comment (realSpok) [reply packages/datasource-customizer/src/decorators/actions/collection.ts:124]: yes.  I will wait for the pages to be available I think

**Review by EnkiP — APPROVED** [review]

**Pull request merged**